data/json/items/gun/44paper.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "colt_army",
+    "type": "GUN",
```

Comment:
Hmm. It would be a bit odd for the reload times of a cap-and-ball pistol to be identical to the template, but it is fine if desired.